docs/conf.py

```diff
@@ -29,7 +30,7 @@
 # -- Project information -----------------------------------------------------
 
 project = 'Fairlearn'
-copyright = '2021, Fairlearn contributors'
+copyright = f'{datetime.now().year}, Fairlearn contributors'
```

Comment:
should this not say `2018-2021, Fairlearn contributors`? I'm really not sure.

Reply:
I'm not sure whether it *has* to, but we could certainly add it (see [Miro's comment](https://github.com/fairlearn/fairlearn/pull/823#issuecomment-857661650)).

Reply:
Yeah @adrinjalali. I will make that change